### PR TITLE
Returned colon with surrounding spaces

### PR DESCRIPTION
**Pull request by DKilkenny**
### Summary

Returned colon with surrounding spaces

### Related Issues

- Resolves #N/A

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

**Review by naylor-b — APPROVED** [review]

File: openmdao/devtools/iprofile_app/iprofile_app.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def _stratify(call_data, sortby='time'):
         # now further group each group by parent, then sort those in descending order
         # by 'sortby'
         depth_groups.append({
-            key: sorted(sub, key=lambda d: d[sortby], reverse=True)
+            key : sorted(sub, key=lambda d: d[sortby], reverse=True)
```
Comment (Kenneth-T-Moore) [inline openmdao/devtools/iprofile_app/iprofile_app.py:63]: Don't think you need this one.

File: openmdao/drivers/scipy_optimizer.py [hunk 2]
```diff
@@ -84,31 +84,31 @@ class ScipyOptimizeDriver(Driver):
...
+    _con_cache : dict
         Cached result of constraint evaluations because scipy asks for them in a separate function.
-    _con_idx: dict
+    _con_idx : dict
         Used for constraint bookkeeping in the presence of 2-sided constraints.
-    _grad_cache: OrderedDict
+    _grad_cache : OrderedDict
         Cached result of nonlinear constraint derivatives because scipy asks for them in a separate
         function.
     _exc_info: 3 item tuple
```
Comment (Kenneth-T-Moore) [inline openmdao/drivers/scipy_optimizer.py:104]: Missed _exc_info

File: openmdao/recorders/sqlite_reader.py [hunk 3]
```diff
@@ -605,7 +605,7 @@ def list_solver_options(self, run_number=0, solver=None, out_stream=_DEFAULT_OUT
                         out_stream.write(f"Run Number: {num}\n")
                         num_header = num
 
-                    out_stream.write(f"    Solver: {name}\n")
+                    out_stream.write(f"    Solver : {name}\n")
```
Comment (Kenneth-T-Moore) [inline openmdao/recorders/sqlite_reader.py:608]: don't change in the prints.

File: openmdao/solvers/linear/petsc_ksp.py [hunk 4]
```diff
@@ -124,9 +124,9 @@ class Monitor(object):
     ----------
     _solver: _solver
```
Comment (Kenneth-T-Moore) [inline openmdao/solvers/linear/petsc_ksp.py:125]: missed _solver

File: openmdao/solvers/solver.py [hunk 5]
```diff
@@ -1024,7 +1024,7 @@ def _iter_get_norm(self):
         """
         Return the norm of the residual.
 
-        Note: This has the side effect of modifying the residual vector in fwd mode
+        Note : This has the side effect of modifying the residual vector in fwd mode
```
Comment (Kenneth-T-Moore) [inline openmdao/solvers/solver.py:1027]: not needed in the note.

File: openmdao/test_suite/groups/parallel_groups.py [hunk 6]
```diff
@@ -124,7 +124,7 @@ def __init__(self):
 
 class DiamondFlat(om.Group):
     """
```
Comment (Kenneth-T-Moore) [inline openmdao/test_suite/groups/parallel_groups.py:126]: Don't think these changes are a good idea either.

File: openmdao/approximation_schemes/approximation_scheme.py [hunk 7]
```diff
@@ -20,18 +20,18 @@ class ApproximationScheme(object):
 
     Attributes
     ----------
-    _approx_groups: list
+    _approx_groups : list
```
Comment (Kenneth-T-Moore) [inline openmdao/approximation_schemes/approximation_scheme.py:23]: Changes to these files are missing:

Missing changes you made to:
  openmdao/code_review/test_lint_attributes.py
  openmdao/test_suite/groups/sin_fitter.py
  openmdao/test_suite/test_examples/beam_optimization/multipoint_beam_group.py 
  openmdao/test_suite/test_examples/beam_optimization/multipoint_beam_stress.py 
  openmdao/test_suite/test_examples/cannonball/test_euler_integration.py

**Review by Kenneth-T-Moore — CHANGES_REQUESTED** [review]

File: openmdao/approximation_schemes/approximation_scheme.py (hunk 7 above)
Comment (DKilkenny) [reply openmdao/approximation_schemes/approximation_scheme.py:23]: Fixed

**Review by Kenneth-T-Moore — APPROVED** [review]

**Pull request merged**